lib/iterator.js

```diff
@@ -156,10 +156,30 @@ class Iterator {
     return new FlatMapIterator(this, mapper, thisArg);
   }
 
+  // Creates an iterator that aggregates elements from each of the iterators.
+  //
+  // Returns an iterator of Arrays where the i-th tuple contains
+  // the i-th element from each of the passed iterators.
+  // The iterator stops when the shortest input iterable is exhausted.
+  // Signature: ...iterators
```

Comment:
No need for the signature change when the default is correct
```suggestion
```

Reply:
The metadoc will generate `zip(iterators)`